src/test/java/tests/ContactTests.java

```diff
@@ -0,0 +1,20 @@
+package tests;
+
+import helpers.LogHelper;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.ContactPage;
+
+public class ContactTests extends BaseTests {
+
+    ContactPage contactPage = new ContactPage();
+
+    @Test(description = "Contact Email contains correct href value which can help to quickly open Outlook Compose Message dialog")
+    public void tc04_CheckValueOfContactEmail() {
+        LogHelper.info("Go to contact tab");
+        contactPage.goToContactTab();
```

Comment:
At beforeMethod, you will stay at homePage by navigating URL. This row is the first action after that.
So it should be `homePage.goToContactTab();`

The same for preCondition in LoginTests 
It should be `homePage.goToLoginTab();`

Reply:
As Tan Le just said, the page name is correct if you don't have HomePage class
just need to update the method name

Reply:
I've just changed method name. Please help me to check it. Thanks